scripts/setup_env_linux_debian.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# setup dev env on ubuntu
+
+# check if sudo permission is not available
+if [ "$EUID" -ne 0 ]; then
+    echo "sudo permission is not available"
+    exit
+fi
+
+# install system-wide dependencies 
+apt-get install python3.6 python-pip
+pip install virtualenv
+
+virtualenv -p $(which python3.6) venv
+source venv/bin/activate
+pip install -r requirements/dev.txt
+
+# create the database using psql and the user postgres
+read -p "Enter username: " username
+read -s -p "Enter password: " password
```

Comment:
What if a confirmation password is added and the two compared to prevent mistakes, because the password is not displayed.